development/Dockerfile

```diff
@@ -2,6 +2,8 @@ ARG PYTHON_VER
 ARG NAUTOBOT_VER
 FROM ghcr.io/nautobot/nautobot-dev:${NAUTOBOT_VER}-py${PYTHON_VER}
 
+RUN pip install -U "setuptools==59.1.1"
```

Comment:
This is a workaround for now as Chatops feature will not run in Docker.  Once celery is updated and the new package is in Nautobot this should be able to be remove.